src/Configuration/src/CloudFoundry/CloudFoundryApplicationOptions.cs

```diff
@@ -7,91 +7,124 @@
 
 namespace Steeltoe.Configuration.CloudFoundry;
 
-public sealed class CloudFoundryApplicationOptions : ApplicationInstanceInfo
+public sealed class CloudFoundryApplicationOptions : IApplicationInstanceInfo
```

Comment:
Changes in this class make it look sooo much more natural here, nice!

Reply:
Totally agree! :)